packages/system/src/Location/ListView/api.ts

```diff
@@ -34,13 +89,27 @@ export const useLocationList = (): UseQueryResult<
     initialSortBy: { key: 'name' },
   });
 
-  const result = useQuery(['locations', 'list', queryParams], async () => {
-    const result = await api.locations({
+  const result = useQuery(['location', 'list', queryParams], async () => {
+    const response = await api.locations({
       sort: [toSortInput(queryParams.sortBy)],
     });
-    const locations = locationsGuard(result);
+    const locations = locationsGuard(response);
     return locations;
```

Comment:
bit of a problem here with sorting. The sorting works fine on the list view, however using this hook in the inbound shipment/`LocationSearchInput` component then the sort gets messed up, as the `useQueryParams` is taking state from the URL, which is for the DetailView - so the sort key is set as `itemName` which is also causing an error when against the demo server API, as `itemName` is not a sort key for location.

Thinking:
- Use another hook for the list which just uses `sortBy`
- Do something about `useQueryParams` and the url state

Unsure which right now

Reply:
ugh. am a little concerned about the `useQueryParams` and url state implementation now, partly due to the navigation getting messed up with it too. not sure at this stage.. two versions of `useQueryParams`? something to indicate whether the current page and the data for it is tied to the URL or not?

Reply:
Ah yes I had forgotten about this .. lucky you're here.

I'm thinking two versions would be easier. 

I'm hesitant, but I think that the benefit for the feature just isn't there right now and that it might just be better to cop it and back out of using the URL like this for now.. 😢 🤷

Reply:
sad. but you are probably right and should just take it out